Source/SLKTextViewController.m

```diff
@@ -2213,7 +2213,6 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView
             _keyboardHeightBeforeDragging = self.keyboardHC.constant;
         }
     }
-    [self dismissKeyboard:YES];
```

Comment:
When we press the return key, scrolled the keyboard and inherently dismissed it. Instead of adding support for this now via a pan gesture, reverting back to what we had